src/samples/sample-cross-fx/README.md

```diff
@@ -1,4 +1,4 @@
-[![Piral Logo](https://github.com/smapiot/piral/raw/master/docs/assets/logo.png)](https://piral.io)
+[![Piral Logo](https://raw.githubusercontent.com/smapiot/piral/master/docs/assets/logo.png)](https://piral.io)
```

Comment:
Should be `https://raw.githubusercontent.com/smapiot/piral/main/docs/assets/logo.png`